src/user_summary/views/main_page.py

```diff
@@ -86,3 +94,28 @@ def load_graphs(request):
                         graphs_object.edits_activity_chart_data,
                          'created_activity_chart_data':
                         graphs_object.created_activity_chart_data})
+
+
+# View that corresponds to editing the profile by logged-in user
```

Comment:
It does not seem to be limited to the logged-in user though.

Reply:
For now it isn't, so that you can test it for multiple useers. Once I've made the edit_profile page, I will turn on the auth part as well.